src/realm/util/interprocess_mutex.hpp

```diff
@@ -144,6 +150,11 @@ inline InterprocessMutex::InterprocessMutex()
 
 inline InterprocessMutex::~InterprocessMutex() noexcept
 {
+#ifdef _WIN32
+    bool b = CloseHandle(m_handle);
```

Comment:
it was by design becuase it would otherwise be a NULL handle. But I added an `if` for readibility